Rpc functionalities (#1687)
oskariorg/oskari-frontend: api/mapping/mapmodule/request/rearrangeselectedmaplayerrequest.md

# RearrangeSelectedMapLayerRequest [rpc]

> Allow user to change selected map layer position (upper/lower) on a map.

Reviewer: ```suggestion
Allow user to change order of map layers in relation to other layers on the map.
```

## Use cases

> - change selected map layer position

Reviewer: ```suggestion
- re-arrange order of layers on the map
```

## Description

> Requests a map layer position to be changed on a map.

Reviewer: ```suggestion
Requests order of a map layer to be changed in relation to other layers on the map.
```

Reviewer: Should document what happens if the index/position is out of bounds. But it's already good that we have atleast some documentation for this.
